packages/shared/types/bookmarks.ts

```diff
@@ -161,6 +167,8 @@ export const zGetBookmarksRequestSchema = z.object({
   // The value is currently not being used, but keeping it so that client can still set it to true for older
   // servers.
   useCursorV2: z.boolean().optional(),
+  sortBy: zSortBy.optional().default("createdAt"),
```

Comment:
Given that this is currently unused, should we just drop it for now to avoid confusion?

Reply:
yes, removed it.